docs/source/redhat-opens-install.rst

```diff
@@ -394,13 +397,7 @@ On each compute node, perform the following steps:
 4. Run ``yum update``. This will bring in Calico-specific updates to the
    OpenStack packages and to ``dnsmasq``.
 
-5. Install build dependencies:
-
-   ::
-
-       yum groupinstall 'Development Tools'
-
-6. Install and configure the DHCP agent on the compute host:
+5. Install and configure the DHCP agent on the compute host:
```

Comment:
It's not so much that it's not required - it's that the etcd part of the instructions has already installed it.